os_stub/cryptlib_openssl/pk/x509.c

```diff
@@ -21,6 +21,9 @@
 /*buffer size to store subject object*/
 #define MAX_SBUJECT_NAME_LEN 0x100
 
+#define LIBSPDM_CRYPTO_ASN1_PARSE_ERROR 0x80
```

Comment:
How do you know this? Is this defined by ASN or OpenSSL? If it's defined by OpenSSL why aren't we using the OpenSSL #defines?

Reply:
> How do you know this? Is this defined by ASN or OpenSSL? If it's defined by OpenSSL why aren't we using the OpenSSL #defines?

Sorry, I can't find a suitable macro in OpenSSL

Reply:
1) define a macro named "OPENSSL_ASN1_ERROR_MASK 0x80", and add link "https://man.openbsd.org/ASN1_get_object.3" to the comment for the MACRO.
2) use "ret & OPENSSL_ASN1_ERROR_MASK" in the code to judge if it is an error.